### PR TITLE
Add configuration variables to jekyll static site

### PR DESCRIPTION
**Pull request by wcyn**
## What Does this PR do?
It replaces the hardcoded jurisdiction information with config variables and leaves the develop branch as a general branch that can be branched from when a new static site is created.

## Major changes made?
- A new branch has been created for the Kenya jurisdiction static site. This means that deploying the Kenya static site would require one to checkout into the kenya branch.
- Two new variables have been added to the env.example file:
`JURISDICTION`  and `JURISDICTION_CODE`. These variables are used in the `deploy.sh` file in order avoid manually editing the file when deploying the static site.
- The `_config.yml` file now has url and jurisdiction variables that are used in the html content. They have replaced the hard-coded values.
- The `build-index.py` file has also been edited to obtain jurisdiction data from the json file

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
       <h2>{{ site.title }}</h2>
       <p>{{ site.description }}</p>
       <p class="footer-govprinters">
-        Gazettes from the <a href="http://kenyalaw.org/kl/" target="_blank" rel="noopener">National Council for Law Reporting (Kenya Law)</a> are re-shared under <a href="https://web.archive.org/web/20170215233726/http://www.kenyalaw.org/kl/index.php?id=2161f" target="_blank" rel="noopener">Creative Commons Attribution - ShareAlike 4.0 International (CC BY-SA 4.0) license</a>.
+        Gazettes from the <a href="https://dds.crl.edu/crldelivery/27040" target="_blank" rel="noopener">Center for Research Libraries: Global Resource Network</a>.
```
Comment (DavidLemayian) [inline _includes/footer.html]: @andela-wcyn We should include the attribution in the config.yml as well for re-use.

File: deploy.sh [hunk 2]
```diff
@@ -14,15 +14,15 @@ export DEPLOY_DIR=".deploy"
...
 
-# Copy the site directory to a temporary location so that modifications we make 
+# Copy the site directory to a temporary location so that modifications we make
 # don't get overwritten by the Jekyll server that is potentially running
 mkdir -p $DEPLOY_DIR
 cp -a _site/. $DEPLOY_DIR
 
-for file in $DEPLOY_DIR/gazettes/KE/*.html; do
-    if [ $file != "$DEPLOY_DIR/gazettes/KE/index.html" ];
+for file in $DEPLOY_DIR/gazettes/NG/*.html; do
```
Comment (DavidLemayian) [inline deploy.sh]: Should this also use `$JURISDICTION_CODE`?

**Review by DavidLemayian — CHANGES_REQUESTED** [review]
Just two slight changes but looks good overall.

File: _includes/footer.html (hunk 1 above)
Comment (wcyn) [reply _includes/footer.html]: True. I'll do that

File: deploy.sh (hunk 2 above)
Comment (wcyn) [reply deploy.sh]: Yea, I missed that one

**Comment by wcyn** [comment]
@DavidLemayian made the requested changes

**Review by DavidLemayian — APPROVED** [review]

**Pull request merged**